bundles/org.openhab.core.io.rest.sse/src/main/java/org/openhab/core/io/rest/sse/SseResource.java

```diff
@@ -120,20 +122,21 @@ public SseResource(@Reference SseItemStatesEventBuilder itemStatesEventBuilder)
         this.executorService = Executors.newSingleThreadExecutor();
         this.itemStatesEventBuilder = itemStatesEventBuilder;
 
-        cleanSubscriptionsJob = scheduler.scheduleWithFixedDelay(() -> {
-            logger.debug("Run clean SSE subscriptions job");
-            OutboundSseEvent outboundSseEvent = sse.newEventBuilder().name("event")
-                    .mediaType(MediaType.APPLICATION_JSON_TYPE).data(new ServerAliveEvent()).build();
-            topicBroadcaster.send(outboundSseEvent);
-        }, 1, 2, TimeUnit.MINUTES);
+        keepaliveJob = scheduler.scheduleWithFixedDelay(() -> {
+            logger.debug("Sending keepalive to SSE connections");
+            OutboundSseEvent keepaliveEvent = sse.newEventBuilder().name("keepalive")
+                    .mediaType(MediaType.APPLICATION_JSON_TYPE).data(new KeepaliveEvent()).build();
+            itemStatesBroadcaster.send(keepaliveEvent);
```

Comment:
This line is new. We have to study if it could impact any sitemap UI. If I am not wrong, the core UI is listening for item state changes.

Reply:
Sorry I'm not following?  If you mean the "keepalive" event, yes it is new, but as a UI you would specifically need to listen to this event type to process the message.  If its a standard browser using the "onMessage" callback, then there would not be an issue, if we have clients who have rolled their own SSE client implementation, then i could see this possibly being an issue.  

My original approach was not to touch the ALIVE message for fear of breaking something sitemap related fyi.

Reply:
I mean you added the line "itemStatesBroadcaster.send(keepaliveEvent);" in addition to the already present line "topicBroadcaster.send(keepaliveEvent);". I am not sure what this line with itemStatesBroadcaster is doing.

Reply:
>I am not sure what this line with itemStatesBroadcaster is doing.

The `/states` endpoint, which the main UI uses,  is using the itemStatesBroadcaster, not the topicBroadcaster , so i'm sending this new message event of type `keepalive` with a payload of the JSON encoded `KeepaliveEvent ` object to those connections as well as the topicBroadcaster which was the original impl.  In terms of the main UI, this will have no affect unless the client starts to listen to this specific event type of "keepalaive",  item events are sent with no event type, which is the default.   I will refactor this again to use the ALIVE type, as to not break existing impls.